src/main/views/waveform.overview.js

```diff
@@ -183,6 +183,9 @@ define([
   };*/
 
   WaveformOverview.prototype.updateRefWaveform = function (time_in, time_out) {
+    if (isNaN(time_in))  throw new Error("WaveformOverview#updateRefWaveform passed a time in that is not a number: " + time_in);
```

Comment:
Is there any reason `time_in` and `time_out` to equal `NaN`? I don't see any code change in our code base which could cause this.

The question could be rephrased as *how useful is it*, because if I understand well, the bug lied in Kinetic.